hkdfchacha20poly1305/hkdfchachapoly.go

```diff
@@ -0,0 +1,87 @@
+package hkdfchacha20poly1305
+
+import (
+	"crypto/cipher"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+	"io"
+
+	"golang.org/x/crypto/chacha20poly1305"
+	"golang.org/x/crypto/hkdf"
+)
+
+var sigma = [4]uint32{0x61707865, 0x3320646e, 0x79622d32, 0x6b206574}
```

Comment:
Add a comment what this sigma means and where the hex nums come from.

Reply:
Woops forgot to delete those, thanks!